Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Use the official Ruby 3.2 image as a base image
+ARG RUBY_VERSION=3.2
+FROM ruby:${RUBY_VERSION}-bullseye
+
+# Set the working directory
+WORKDIR /usr/src/app
+
+# Install system packages
+RUN apt-get update -qq && \
+    apt-get install -y default-mysql-client vim && \
```

Comment:
Adding `vim` isn't necessary, but it sure makes things easier to debug in container